modules/test/playwright/pages/object-web/object-layout/ObjectLayoutsPage.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {FrameLocator, Locator, Page} from '@playwright/test';
+
+import {ViewObjectDefinitionsPage} from '../ViewObjectDefinitionsPage';
+
+export class ObjectLayoutsPage {
+	readonly addBlock: Locator;
+	readonly addField: Locator;
+	readonly addObjectLayoutButton: Locator;
+	readonly addTab: Locator;
+	readonly iframeLocator: FrameLocator;
+	readonly layoutNameBox: Locator;
```

Comment:
Hey @vngoliveira, for Box do you want say Modal? I didn't quite understand that term. However,  We can just use `layoutNameInput`. 

I understand the absence of the object prefix for these cases. When we are working with locators, we must be faithful to what is being rendered on the screen 😄